src/oidcop/oidc/token.py

```diff
@@ -280,6 +277,8 @@ def process_request(self, req: Union[Message, dict], **kwargs):
             _resp["id_token"] = _idtoken.value
 
         token.register_usage()
+        if kwargs.get("revoke_refresh_on_issue", False):
```

Comment:
I think that this should be on the class (for global configuration) and on the client (for per-client configuration). We shouldn't have to pass it to the function every time. Maybe keep this and add the other 2 methods?

Also I think `True` is a better default value.